### PR TITLE
engine/CMakeLists.txt: Add libglu1-mesa as bionic install dependency

### PR DESCRIPTION
**Pull request by stephengtuggy**
Thank you for submitting a pull request and becoming a contributor to the Vega Strike Core Engine.

Please answer the following:

Code Changes:
- [ ] Have the PR Validation Tests been run? See https://github.com/vegastrike/Vega-Strike-Engine-Source/wiki/Pull-Request-Validation
- [ ] This is a documentation change only
- [x] This is a build system/installer change only

Issues:
- Please list any related issues

Purpose:
- What is this pull request trying to do? **Add a missing installer dependency on Ubuntu 18.04 LTS "bionic"**
- What release is this for? **0.7.0**
- Is there a project or milestone we should apply this to? **0.7.x**

Remarks:
**Same as #375 , only for Ubuntu bionic**

### CODE REVIEW

File: engine/CMakeLists.txt [hunk 1]
```diff
@@ -1296,7 +1296,7 @@ ELSEIF (CMAKE_SYSTEM_NAME STREQUAL Linux)
                 ELSE (USE_PYTHON_3)
                     SET(CPACK_DEBIAN_PACKAGE_DEPENDS "libpython2.7")
                 ENDIF (USE_PYTHON_3)
-                SET(CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_DEPENDS}, libjpeg62, libpng16-16, freeglut3, libgtk-3-0, libvorbis0a, libopenal1, libsdl-gfx1.2-5, xdg-utils, libgl1, libopengl0, libboost-python1.65.1, libboost-log1.65.1")
+                SET(CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_DEPENDS}, libjpeg62, libpng16-16, freeglut3, libgtk-3-0, libvorbis0a, libopenal1, libsdl-gfx1.2-5, xdg-utils, libgl1, libglu1-mesa, libopengl0, libboost-python1.65.1, libboost-log1.65.1")
```
Comment (BenjamenMeyer) [inline engine/CMakeLists.txt:1299]: Is there a space after lib and before gtk?
Hard to tell from my phone.

**Review by BenjamenMeyer — COMMENTED** [review]
Question...

File: engine/CMakeLists.txt (hunk 1 above)
Comment (stephengtuggy) [reply engine/CMakeLists.txt:1299]: In `libgtk-3-0`? No, there isn't. No space

**Review by BenjamenMeyer — APPROVED** [review]
Lgtm

**Pull request merged**